src/plugins/js-eval/jsEvalPlugin.js

```diff
@@ -1,19 +1,57 @@
 const { exec } = require('child_process');
 const crypto = require('crypto');
 
-const jsEvalPlugin = ({ mentionUser, respond, message }) => {
+const children = new Set();
+
+const jsEvalPlugin = async ({ mentionUser, respond, message, selfConfig, log }) => {
   if (!message.startsWith('n>')) return;
   const code = message.slice(2);
   const childId = `jseval-${crypto.randomBytes(8).toString('hex')}`;
 
-  const child = exec(`docker run --rm -i --net=none --name=${childId} devsnek/js-eval`, { timeout: 5000 }, (err, stdout = '') => {
```

Comment:
I thought that by using exec(..., {timeout: ..}) it would abstract all this timeout work, but I guess you still want to cleanup the processes?

Reply:
It wasn't killing them. Maybe because we sent SIGKILL to the `docker run` process and the container isn't owned by the `docker run` process, it's owned by the daemon. We can kill the process violently, and politely request that docker kills the container.